dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -1,58 +1,427 @@
+								index === boostedIndex ? '75%' : '25%'
+							}
+							showDivider={index > 0}
+							showTopMarginWhenStacked={index > 0}
+						>
+							<FrontCard
+								trail={card}
+								containerPalette={containerPalette}
+								showAge={showAge}
+								supportingContent={card.supportingContent}
+								headlineSize={
+									index === boostedIndex ? 'large' : 'medium'
+								}
+								trailText={
+									index === boostedIndex
+										? card.trailText
+										: undefined
+								}
+								imageUrl={card.image}
+								imagePosition={
+									index === boostedIndex ? 'right' : 'top'
+								}
+								imagePositionOnMobile={
+									index === boostedIndex ? 'top' : 'left'
+								}
+								imageSize={
+									index === boostedIndex ? 'large' : 'small'
+								}
+							/>
+						</LI>
+					);
+				}
+
+				// Unboosted primaries split 50/50 when there are two
+				// or take up a 'huge' 100% role when theres just one
+				return (
+					<LI
+						key={card.url}
+						padSides={true}
+						percentage={primaries.length === 1 ? '100%' : '50%'}
+						showDivider={index > 0}
+						showTopMarginWhenStacked={index > 0}
+					>
+						<FrontCard
+							trail={card}
+							containerPalette={containerPalette}
+							showAge={showAge}
+							supportingContent={card.supportingContent}
+							headlineSize="large"
+							imageUrl={card.image}
+							imagePosition={'top'}
+							imagePositionOnMobile={'top'}
+						/>
+					</LI>
+				);
+			})}
+		</UL>
+	);
+};
+
+/**
+ * When the first 'big' is boosted & there is atleast 1 more big we switch up the layout a little -
+ *
+ * The 'standards' section (what this component renders), consists of 2 columns, rather than a wrapped row,
+ * where the first big (all other 'bigs' fallback to standards) sits in the first slot of the left hand column,
+ * and it gets a top image (left on mobile), and is followed by up to the next 2 standards.
+ * The right hand column will take (up to) the remaining 5 standards. Any more standards will not be rendered.
+ *
+ */
+const ThirdBoostedPlusBig = ({
+	standards,
+	big,
+	showAge,
+	containerPalette,
+}: {
+	standards: TrailType[];
+	big: TrailType;
+	showAge?: boolean;
+	containerPalette?: DCRContainerPalette;
+}) => {
+	// 'col1' has 1 big & up to 2 standards
+	const standardsCol1 = standards.splice(0, 2);
+	// 'col2' has up to 5 standards
+	const standardsCol2 = standards.splice(0, 5);
+
+	const manualUlStyles = css`
+		position: relative;
+		display: flex;
+		flex-direction: column;
+		flex-basis: 50%;
+		flex-grow: 1;
+		${until.tablet} {
+			flex-basis: 100%;
+		}
+	`;
+
+	/**
+	 * The row & columns in this component are a little outside of the standard of what the standard
+	 * LI and UL components we have support - it made more sense to manually create the CSS for this use case
+	 * than to expand & muddy the APIs of both components to support this layout type.
+	 */
+	return (
+		<li
```

Comment:
I suppose the risk here is any changes to LI or UL will not be picked up by this code. This makes me a little uncomfortable in that I feel that we're breaching a contract here. At the moment, every single Card list is put through a UL/LI combo but now we're adding an exception.

I can appreciate though that these abstractions are very heavy and changing them isn't easy